src/frontend-samples/display-styles-sample/Styles.ts

```diff
@@ -153,7 +144,7 @@ export const displayStyles: DisplayStyle[] = [{
 }, {
   name: "Outdoorsy",
   environment: {
-    sky: { display: true, twoColor: false, groundColor: 8228728, zenithColor: 16741686, nadirColor: 3880, skyColor: 16764303 },
+    sky: { display: true, twoColor: false, groundColor: 9741199, nadirColor: 5464143, skyColor: 16764303, zenithColor: 16741686 },
```

Comment:
Can you define `defaultSky` and just use `sky: { ...defaultSky }` instead of copy-pasting?